infrastructure/main.tf

```diff
@@ -130,6 +130,9 @@ data "vault_generic_secret" "bulkScanProcessorTests" {
   path = "secret/${var.vault_section}/ccidam/service-auth-provider/api/microservice-keys/bulk-scan-processor-tests"
 }
 
+data "vault_generic_secret" "barApi" {
+  path = "secret/${var.vault_section}/ccidam/service-auth-provider/api/microservice-keys/bar-api"
```

Comment:
@attila-kiss-hmcts Have you added all the secrets to the hashicorp vault in all environments ?

Reply:
@nitinprabhu yes we added the "bar-api" key in dev/test//preprod/demo and prod tactical